src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -279,58 +285,84 @@ struct rr_connectivity_data {
   glb_lb_policy *glb_policy;
 };
 
+static bool process_serverlist(const grpc_grpclb_server *server,
```

Comment:
The function as been completely rewritten and now does process the whole serverlist.